datacenter/dtr/2.2/guides/release-notes/index.md

```diff
@@ -12,6 +12,25 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](../admin/upgrade.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.2.8
+
+(24 August 2017)
+
+**New features**
+
+* Added UI support to configure if users should be redirected to the storage
```

Comment:
It's a registry option that was added to the UI. It's not a new feature. The UI for it is new. I guess it depends on your definition of "new feature"